docs/src/components/ComponentPlayground/createHookGenerator.ts

```diff
@@ -41,7 +41,7 @@ const createHookGenerator = (options: KnobGeneratorOptions): null | KnobDefiniti
   if (process.env.NODE_ENV !== 'production') {
     if (!Component) {
       throw new Error(
-        `Cannot find an export for "${componentInfo.displayName}", please check that it is exported from "@stardust-ui/react"`,
+        `Cannot find an export for "${componentInfo.displayName}", please check that it is exported from "@fluentui/react"`,
```

Comment:
Seems that these should have taken a dependency on `package.json` and just linked to the `name`.  Meh.

Reply:
Oh I see, but heh, who would have thought we'd rename a package :)

Reply:
I suppose it would be a good change - but I want to keep the diffs pretty plain for now. Maybe a good todo later?